<p>Javax persistence annotations should be placed on accessors</p>
<h2>Noncompliant Code Example</h2>
<pre>
    @Id private Long id; // Noncompliant
</pre>
<h2>Compliant Solution</h2>
<pre>
    @Id
    public Long getId() {
        return id;
    }
</pre>
